FuelLabs/fuels-wallet
# feat: add error reporting tool using sentry

This PR;
- Improves error handling for CRX
- Improve error handling UI to be less dependent on the application UI
- Add checking on database to report also CRX errors / crashes
- Fix the UI of the error handling screen related to the top items
- Change error reporting to use Sentry instead of email

How to test;
- Install the wallet extension;
- Open the service worker console, via the extension manager page link service worker.
- Execute the following command to dispatch a error `self.dispatchEvent(new ErrorEvent('error'));`
- Open the wallet, it should should the error reporting screen.
